packages/scrapers-v2/src/lexer/types.ts

```diff
@@ -0,0 +1,109 @@
+/**
```

Comment:
these comments & the readme look so nice

Reply:
yay!